packages/fiori-freestyle/src/index.ts

```diff
@@ -9,13 +9,13 @@ import {
     WorklistSettings,
     ListDetailSettings,
     Template,
-    Package,
-    OdataVersion
+    Package
 } from '@sap/open-ux-tools-types';
 import { TemplateType } from '@sap/open-ux-tools-types'; // This is an enum dont import as type, we lose runtime values
 import { UI5Config } from '@sap/ux-ui5-config';
 import { getPackageTasks } from '@sap/open-ux-tools-common';
 import { getUI5Libs } from './data/ui5Libs';
+import cloneDeep from 'lodash.clonedeep';
```

Comment:
```suggestion
import cloneDeep from 'lodash/cloneDeep';
```

Reply:
@devinea, I could not find any indication that `lodash.deepClone` is the older approach. They still seem to have the link to the per method packages in their doco: https://www.npmjs.com/search?q=keywords:lodash-modularized

Looking at the way they build the packages (https://lodash.com/custom-builds) and also comparing the dates of the last changes (2 years ago), both `lodash.deepClone` and `lodash/deepClone` seem to be the same implemenation.

Package sizes are different though. `lodash.deepClone` is 45kb while `lodash` is 1.4 MiB. Treeshaking in our webpack config might pare it down to what's needed but do you think we should still use `lodash/deepClone`?

Reply:
They also have this: https://www.npmjs.com/package/lodash-es